### PR TITLE
chore: translation for unauthorized workflow error

### LINKED ISSUE #7421: Add missing translation
https://github.com/calcom/cal.com/blob/0ec71e52eff2ac8bdfd4bdd4616c6704f9f5aaeb/packages/features/ee/workflows/components/EventWorkflowsTab.tsx#L69

```javascript

        const message = `${err.statusCode}: ${err.message}`;
        showToast(message, "error");
      }
      if (err.data?.code === "UNAUTHORIZED") {
        // TODO: Add missing translation
        const message = `${err.data.code}: You are not authorized to enable or disable this workflow`;
        showToast(message, "error");
      }
    },
  });


```

### PR DESCRIPTION
**Pull request by iamsriranga**
## What does this PR do?

Fixes https://github.com/calcom/cal.com/issues/7421

## Type of change

- Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?

- [ ] Test A
- [ ] Test B

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't performed a self-review of my own code and corrected any misspellings
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/features/ee/workflows/components/EventWorkflowsTab.tsx [hunk 1]
```diff
@@ -70,9 +70,7 @@ const WorkflowListItem = (props: ItemProps) => {
         showToast(message, "error");
       }
       if (err.data?.code === "UNAUTHORIZED") {
-        // TODO: Add missing translation
-        const message = `${err.data.code}: You are not authorized to enable or disable this workflow`;
```
Comment (hariombalhara) [inline packages/features/ee/workflows/components/EventWorkflowsTab.tsx:74]: Let's make this string a dynamic one that can accept err.data.code as variable e.g. like here https://github.com/calcom/cal.com/blob/94b3945ffc423ba3f5770fb7f8f16a3fbe181828/apps/web/public/static/locales/en/common.json#L1707

**Review by hariombalhara — CHANGES_REQUESTED** [review]
Left a comment.

File: packages/features/ee/workflows/components/EventWorkflowsTab.tsx (hunk 1 above)
Comment (saideepesh000) [reply packages/features/ee/workflows/components/EventWorkflowsTab.tsx:74]: sure, will make the changes

File: packages/features/shell/Shell.tsx [hunk 2]
```diff
@@ -299,19 +299,19 @@ function UserDropdown({ small }: { small?: boolean }) {
                 />
               }
               {!user.away && (
-                <div className="absolute bottom-0 right-0 h-3 w-3 rounded-full border-2 border-muted bg-green-500" />
+                <div className="border-muted absolute bottom-0 right-0 h-3 w-3 rounded-full border-2 bg-green-500" />
```
Comment (saideepesh000) [inline packages/features/shell/Shell.tsx:302]: are these linting changes?

**Comment by saideepesh000** [comment]
> Left a comment.

@hariombalhara made the changes

**Review by hariombalhara — APPROVED** [review]
Thankyou 🙏

**Comment by hariombalhara** [comment]
There are some merge conflicts  @saideepesh000

**Pull request merged**